### PR TITLE
717 test production mode in integration test

### LINKED ISSUE #717: Test production mode in integration test
Currently, we only test ManagementPortal in dev mode in Github actions. This does not test the PostgreSQL integration. Ideally, we add a step to the Github Actions docker job to run the created docker image with docker-compose and test if it functions. A minimal implementation tests if ManagementPortal becomes healthy.

An example would be the first two services in https://github.com/RADAR-base/radar-app-config/blob/master/docker-compose.yml.

### PR DESCRIPTION
**Pull request by Bdegraaf1234**
Description: Include production testing in the github actions workflow to test postgres integration

Fixes #717

#### Checklist:
- [x] The [Main workflow](https://github.com/RADAR-base/ManagementPortal/actions/workflows/main.yml) has succeeded
- [x] The [Gatling tests](https://github.com/RADAR-base/ManagementPortal#other-tests) have passed
- [x] I have logged into the portal running locally with default admin credentials
- [x] I have updated the README files if this change requires documentation update
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: .github/workflows/pre-release.yml [hunk 1]
```diff
@@ -51,14 +51,12 @@ jobs:
...
+      - name: Run full production end to end tests
         run: |
-          cp src/test/resources/config/keystore.p12 src/main/resources/config/keystore.p12
+          docker-compose -f src/main/docker/postgresql.yml up -d
           ./gradlew bootRun &>mp.log </dev/null &
-          yarn run wait-for-managementportal
-          ./gradlew generateOpenApiSpec
-          yarn e2e
-          ./gradlew --stop
+          yarn run e2e-prod
```
Comment (yatharthranjan) [inline .github/workflows/pre-release.yml:58]: do you need to wait here for the services to startup?

File: README.md [hunk 2]
```diff
@@ -68,15 +68,14 @@ You must install and configure the following dependencies on your machine to run
...
-memory database and ManagementPortal. 
+memory database and ManagementPortal. An angular live development server to access the managementportal can be started using the `yarn start` command (see [Development](#development)).
 6. You can log in to the application using `admin:admin`. Please don't forgot to change the password of `admin`, if you are using the application on production environment.
 
 
 |                                  | Development     | Production                        |
 |----------------------------------|-----------------|-----------------------------------|
 | Database type                    | In-memory       | Postgres                          |
 | Demo data loaded                 | Yes             | No                                |
-| Context path of the application  | `/`             | `/managementportal`               |
```
Comment (yatharthranjan) [inline README.md:79]: has this changed?

File: src/main/resources/config/application-prod.yml [hunk 3]
```diff
@@ -19,12 +19,12 @@ spring:
         livereload:
             enabled: false
     datasource:
-        type: com.zaxxer.hikari.HikariDataSource
-        driverClassName: org.postgresql.Driver
-        url: jdbc:postgresql://localhost:5432/managementportal
-#        name: managementportal
-        username: <username>
-        password: <password>
+      type: com.zaxxer.hikari.HikariDataSource
```
Comment (yatharthranjan) [inline src/main/resources/config/application-prod.yml:22]: seems indentation is not consistent with rest of the file?

File: src/main/resources/config/application-prod.yml [hunk 4]
```diff
@@ -53,8 +53,6 @@ spring:
             starttls.enable: false
     thymeleaf:
         cache: true
-    liquibase:
-        contexts: prod
```
Comment (yatharthranjan) [inline src/main/resources/config/application-prod.yml:57]: does this need to be removed? Would it not be overridden by the `e2e-prod-test` profile?

**Review by yatharthranjan — COMMENTED** [review]
Thanks @Bdegraaf1234, looks good. Just a few questions (apologies if some are naive, i am looking at MP code after a long time).

File: .github/workflows/pre-release.yml (hunk 1 above)
Comment (Bdegraaf1234) [reply .github/workflows/pre-release.yml:58]: the yarn e2e command has a built in wait period :)

File: README.md (hunk 2 above)
Comment (Bdegraaf1234) [reply README.md:79]: it has, see #733

File: src/main/resources/config/application-prod.yml (hunk 3 above)
Comment (Bdegraaf1234) [reply src/main/resources/config/application-prod.yml:22]: You're right, I fixed it, and now its in line with the .editorconfig as well

File: src/main/resources/config/application-prod.yml (hunk 4 above)
Comment (Bdegraaf1234) [reply src/main/resources/config/application-prod.yml:57]: I moved it to the apllication.yml, the reasoning being that it is the default unless overriddedn by one of the dev profiles. Don't feel super strongly on this.

**Review by peyman-mohtashami — DISMISSED** [review]
LGTM

**Review by peyman-mohtashami — APPROVED** [review]

**Pull request merged**